packages/malloy/src/malloy.ts

```diff
@@ -223,12 +223,14 @@ export class Malloy {
     model,
     refreshSchemaCache,
     noThrowOnError,
+    replaceMaterializedReferences,
   }: {
     urlReader: URLReader;
     connections: LookupConnection<InfoConnection>;
     parse: Parse;
     model?: Model;
-  } & CompileOptions): Promise<Model> {
+  } & CompileOptions &
+    PrepareResultOptions): Promise<Model> {
```

Comment:
When asking the Malloy object to compile something might want to pass in `{ replaceMaterializedReferences: true }`, because an SQL block may contain queries which should have their heads materialized.